src/main/java/views/add_edit_category_views/CategoryC.java

```diff
@@ -0,0 +1,51 @@
+package views.add_edit_category_views;
+
+import entities.Category;
+import entities.EntityException;
+import entities.SessionStorage;
+import use_cases.add_edit_category_use_case.CategoryIB;
+import use_cases.add_edit_category_use_case.CategoryID;
+import use_cases.add_edit_category_use_case.CategoryOD;
+
+public class CategoryC {
```

Comment:
Missing class level JavaDoc

Reply:
The JavaDoc is cut off mid-word 😭

Reply:
Also, I think Kat has mentioned some where else, that it is not clear to say "programme". Maybe say more specifically what the class does like "CategoryC is the controller class in Add/Edit Category use case. This class creates the input data class, and calls input boundary interface method to add/edit category and get output data.